src/script/lua_api/l_object.cpp

```diff
@@ -914,6 +914,58 @@ int ObjectRef::l_get_yaw(lua_State *L)
 	return 1;
 }
 
+// set_pitch(self, radians) (X-AXIS)
+int ObjectRef::l_set_pitch(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	LuaEntitySAO *co = getluaobject(ref);
+	if (co == NULL) return 0;
+	float pitch = luaL_checknumber(L, 2) * core::RADTODEG;
+	// Do it
```

Comment:
Very helpful. Could you please remove this one in all your functions?

Reply:
I'll need some more clarification here. "this one" what? Remove that line? Then the pitch and roll will never be assigned. It is exactly the same way that yaw is set (already implemented before I did anything). I'm sorry you have to deal with my inexperience, but I'm trying my best to get this implemented since it will break things and 0.5.0 is the best time to do that. Thanks again.

Reply:
I think this is sarcasm referring to the 'Do it' comment, which is unnecessary.

Reply:
Ah, I see. I was just keeping the comments the same since it was already in the yaw function. Will remove it. Wasn't my doing, blame whoever wrote the first function (and me, I should have thought the same).

Reply:
Ah ok, yes current code sometimes has stuff we now prefer to change.

Reply:
Again no need to remove these comments from unrelated functions.